install.sh

```
@@ -1238,92 +1281,310 @@ update_zshrc() {
+            return 0
+        fi
+    done
+    return 1
+}
+
+# Load a specific preset by file name (without extension) - CLI usage
+load_preset() {
+    local preset_id="$1"
+    local preset_file="$SCRIPT_DIR/presets/${preset_id}.sh"
+    if [ -f "$preset_file" ]; then
+        source "$preset_file"
```

Shellcheck (0.11.0)

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)